### PR TITLE
fix: add check for os shell in RNkeys.gradle file

### PR DESCRIPTION
**Pull request by moses-20**
## Summary
Fixes [an issue](https://github.com/numandev1/react-native-keys/issues/43#issuecomment-1683553366) where build fails on windows.

Added a check to determine os shell before executing `exportCommand` in the loadKeys function.

## Changelog

[CATEGORY] [TYPE] - Message

### CODE REVIEW

File: android/RNKeys.gradle [hunk 1]
```diff
@@ -83,7 +83,15 @@ def loadKeys() {
       def nodeModulesDir = getNodeModulesDir()
       exportCommand = exportCommand + nodeModulesDir + "/node_modules/react-native-keys/keysAndroid.js"
     }
-    def proc = ["/bin/sh", "-c", exportCommand].execute()
+
```
Comment (numandev1) [inline android/RNKeys.gradle:86]: I think, we also need to replace `export` with `set` as cmd dont know about `export`

**Review by numandev1 — CHANGES_REQUESTED** [review]

File: android/RNKeys.gradle (hunk 1 above)
Comment (moses-20) [reply android/RNKeys.gradle:86]: Yes, that might be correct but this works currently.

Comment (numandev1) [reply android/RNKeys.gradle:86]: if you don't replace `export` with `set` then example of this project wont work, can you please add conditional export and set, so i can merge this PR?

Comment (moses-20) [reply android/RNKeys.gradle:86]: Alright I will

Comment (numandev1) [reply android/RNKeys.gradle:86]: @moses-20 ?

**Pull request closed without merging**